models.py

```diff
@@ -8,7 +9,14 @@
 
 import seed
 
-engine = create_engine(os.environ["DATABASE_URL"])
+# @todo: Refactor: handle this in config.py instead.
+vcap_services = json.loads(os.environ.get('VCAP_SERVICES', '{}'))
```

Comment:
I'd recommend sticking with the `os.environ["DATABASE_URL"]` here. When a [database service binds](https://docs.cloud.gov/apps/databases/), it sets `DATABASE_URL` as an env var, so parsing `VCAP_SERVICES` should only be necessary if you're trying to access other metadata about the service or credentials from a user-provided service. This also makes things a little easier when doing local development.